code/API_definitions/S2CVPN4(1).yaml

```
@@ -0,0 +1,936 @@
+openapi: "3.0.3"
+info:
+  title: Site to Cloud VPN API 

```

Title shouldn't contain "API"
```suggestion
  title: Site to Cloud VPN
```